spec/requests/get_five_random_trump_quotes_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.describe 'GET /quotes/:id', type: :request do
+  describe 'with "random" as query parameters' do
+    before do
+      get '/quotes/random'
+    end
+
+    it 'is expected to return a quote' do
+      expected_response_body = {
+        "date": 'October 2016',
+        "source": 'https://twitter.com/realDonaldTrump/status/787699930718695425',
+        "quote_of_the_day": 'The election is absolutely being rigged by the dishonest and distorted media pushing Crooked Hillary - but also at many polling places - SAD'
```

Comment:
Again, I wonder about the naming here...

Reply:
This is fixed as mentioned!